plugins/modules/notification/discord.py

```diff
@@ -0,0 +1,172 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2021, Christian Wollinger <cwollinger@web.de>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+from ansible.module_utils.urls import fetch_url
+from ansible.module_utils.basic import AnsibleModule
+__metaclass__ = type
+
+DOCUMENTATION = '''
+---
+module: discord
+short_description: Send discord messages
+description:
+    - Send a messages to a discord channel via the discord webhook API.
+author: Christian Wollinger (@cwollinger)
+notes:
+  - Find the API documentation for Discord API here: U(https://discord.com/developers/docs/resources/webhook#execute-webhook).
+options:
+  webhook_id:
+    description:
+      - The webhook id
+      - Format from discord webhook URL: C(/webhooks/{webhook.id}/{webhook.token})
```

Comment:
```suggestion
      - Format from discord webhook URL: C(/webhooks/{webhook.id}/{webhook.token}).
```
Please use punctuation in the description lines.